ACA-Py / AFJ Tests are failing

19 of the tests are suddenly failing -- not sure what the issue(s) is(are). For example - Acme is ACA-Py

```
AssertionError: FAILED SUB-STEP: When "Acme" creates a new schema Substep info: Assertion Failed: resp_status 500 is not 200; Extra data: line 1 column 4 (char 3) Traceback (of failed substep): File "/usr/local/lib/python3.7/site-packages/behave/model.py", line 1329, in run match.run(runner.context) File "/usr/local/lib/python3.7/site-packages/behave/matchers.py", line 98, in run self.func(context, *args, **kwargs) File "features/steps/0036-issue-credential.py", line 86, in step_impl assert resp_status == 200, f"resp_status {resp_status} is not 200; {resp_text}"
```

See this page for the full list of errors: https://allure.vonx.io/api/allure-docker-service/projects/acapy-b-javascript/reports/416/index.html#categories/8fb3a91ba5aaf9de24cc8a92edc82b5d

@ianco -- can you please take a first look at this one?

--- a/aries-backchannels/acapy/acapy_backchannel.py
+++ b/aries-backchannels/acapy/acapy_backchannel.py
@@ -656,6 +656,18 @@ class AcaPyAgentBackchannel(AgentBackchannel):
             agent_operation = "/schemas"
             log_msg(agent_operation, command.data)
 
+            # Check if schema id already exists
+            schema_name = command.data.get("schema_name")
+            schema_version = command.data.get("schema_version")
+            (resp_status, resp_text) = await self.admin_GET(
+                "/schemas/created",
+                params={"schema_version": schema_version, "schema_name": schema_name},
+            )
+            resp_json = json.loads(resp_text)
+            if len(resp_json["schema_ids"]) > 0:
+                schema_id = resp_json["schema_ids"][0]
+                return (200, json.dumps({"schema_id": schema_id}))
+
             (resp_status, resp_text) = await self.admin_POST(
                 agent_operation, command.data
             )
@@ -668,6 +680,26 @@ class AcaPyAgentBackchannel(AgentBackchannel):
             # POST operation is to create a new cred def
             agent_operation = "/credential-definitions"
             log_msg(agent_operation, command.data)
+
+            # Check if credential definition id already exists
+            schema_id = command.data.get("schema_id")
+            tag = command.data.get("tag")
+            (resp_status, resp_text) = await self.admin_GET(
+                "/credential-definitions/created",
+                params={
+                    "schema_id": schema_id,
+                },
+            )
+            resp_json = json.loads(resp_text)
+            if len(resp_json["credential_definition_ids"]) > 0:
+                # need to check the 'tag' value
+                for cred_def_id in resp_json["credential_definition_ids"]:
+                    cred_def_id_parts = cred_def_id.split(":")
+                    if tag == cred_def_id_parts[4]:
+                        return (
+                            200,
+                            json.dumps({"credential_definition_id": cred_def_id}),
+                        )
 
             (resp_status, resp_text) = await self.admin_POST(
                 agent_operation, command.data

